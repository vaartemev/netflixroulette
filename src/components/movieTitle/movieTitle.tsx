import React from 'react';

import './movieTitle.scss';

interface Props {
  className: string;
  title: string;
}

export const MovieTitle = ({ className, title }: Props) => {
  return <h2 className={className}>{title}</h2>;
};
