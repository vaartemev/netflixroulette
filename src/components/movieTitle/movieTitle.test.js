import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { MovieTitle } from './movieTitle';

describe('Movie Title', () => {
  const props = {
    title: 'string2',
    className: 'string'
  };

  const output = shallow(<MovieTitle {...props} />);

  it('renders correctly', () => {
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
