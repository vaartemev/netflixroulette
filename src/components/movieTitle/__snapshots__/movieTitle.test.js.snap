// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Title renders correctly 1`] = `
<h2
  className="string"
>
  string2
</h2>
`;
